compiler/damlc/tests/daml-test-files/ExportList.EXPECTED.rst

```diff
@@ -3,6 +3,105 @@
 Module ExportList
 -----------------
 
+Templates
+^^^^^^^^^
+
+.. _type-exportlist-template0-22237:
+
+**template** `Template0 <type-exportlist-template0-22237_>`_
+
+  .. list-table::
+     :widths: 15 10 30
+     :header-rows: 1
+  
+     * - Field
+       - Type
+       - Description
+     * - tfield0
+       - Party
+       - 
+     * - tfield0'
+       - Party
+       - 
+  
+  + **Choice External\:Archive**
+    
+  
+  + **Choice Choice0**
+    
+
+.. _type-exportlist-template1-34256:
+
+**template** `Template1 <type-exportlist-template1-34256_>`_
+
+  .. list-table::
+     :widths: 15 10 30
+     :header-rows: 1
+  
+     * - Field
+       - Type
+       - Description
+     * - tfield1
+       - Party
+       - 
+     * - tfield1'
+       - Party
+       - 
+  
+  + **Choice External\:Archive**
+    
+  
+  + **Choice Choice1**
+    
+
+.. _type-exportlist-template2-3915:
+
+**template** `Template2 <type-exportlist-template2-3915_>`_
+
+  .. list-table::
+     :widths: 15 10 30
+     :header-rows: 1
+  
+     * - Field
+       - Type
+       - Description
+     * - tfield2
+       - Party
+       - 
+     * - tfield2'
+       - Party
+       - 
+  
+  + **Choice External\:Archive**
+    
+  
+  + **Choice Choice2**
+    
+
+.. _type-exportlist-template3-57838:
+
+**template** `Template3 <type-exportlist-template3-57838_>`_
+
+  .. list-table::
+     :widths: 15 10 30
+     :header-rows: 1
+  
+     * - Field
+       - Type
+       - Description
+     * - tfield3
+       - Party
+       - 
+     * - tfield3'
+       - Party
+       - 
+  
+  + **Choice External\:Archive**
+    
+  
+  + **Choice Choice3**
+    
+
 Typeclasses
 ^^^^^^^^^^^
 
```

Comment:
BTW further down this output there is documentation for an expoerted `Constr3` (I cannot place the comment on any relevant line because none of them changed).
The rst contains a _link_ and link target for the constructor. These constructor link targets are syntactically incorrect in rst because of their indentation. Also, the link is outright _useless_ IMHO - how would anyone/anything want to link to a constructor when all documentation is about _types_ not values?

Would be good to remove the constructor links altogether.

Reply:
Disagree on both counts. The links are syntactically correct  And they're not useless at all. There's lots of examples of such links in the DAML standard library docs, by the way.